test/testpath.cpp

```diff
@@ -38,6 +38,20 @@ class TestPath : public TestFixture {
         TEST_CASE(is_c);
         TEST_CASE(is_cpp);
         TEST_CASE(get_path_from_filename);
+        TEST_CASE(isMatchingFilter);
+    }
+
+    void isMatchingFilter() const
+    {
+        ASSERT(Path::isMatchingFilter("c:\\foo\\bar\\os_linux.c", "bar"));
```

Comment:
I'd recommend that in these paths we replace `\\` with `/` .. internally in cppcheck we always use the path separator `/` even in windows.